support-docs/README.md

```diff
@@ -107,7 +109,131 @@ Table below shows the list of properties used to describe pins
 |polarity|Whether the active state of a pin is high or low|String| |
 
 
+#### 3.7	Graph Specifications
+Datasheets include additional information through figures, graphs. Examples include but are not limited to efficiency vs load curves, power derating curves. A graph object and a curve object are specified to capture that information in the digital datasheet. The graph object focuses on defining the type of data captured, including title and axis information. The curve object focuses on capturing the data. Multiple curve objects may be included in a graph object to capture relationships between variables under different conditions. Examples include load current vs efficiency curves for different input voltages.Properties used to specify a curve object are included in the table below:
+|Property|Description|Data Type|Required?|
+|:----|:----|:----|:----|
+|label|Description of the data in a curve|string|Yes|
+|xData|x value of data being plotted|array|Yes|
+|yData|y value of data being plotted|array|Yes|
 
+Properties used to specify a graph object are included below:
+|Property|Description|Data Type|Required?|
+|:----|:----|:----|:----|
+|title|Title of a graph|string|Yes|
+|xUnits|x-axis units|string|Yes|
+|xLabel|x-axis title|String|Yes|
+|yUnits|y-axis units|string|Yes|
+|yLabel|y-axis title|String|Yes|
+
+#### 3.8	Physical Package Specifications
+The digital datasheet has specifications to capture the first level physical package specifications. Properties used to specify a package are included below:
+|Property|Description|Data Type|Required?|
+|:----|:----|:----|:----|
+|length|Length of the package|Number|Yes|
+|width|Width of the package|Number|Yes|
+|height|Height of the package|Number|Yes|
+|dimensionUnit|Unit used to describe the package dimensions|String|Yes|
+|standardPackageSize|Standard name used to designate the package size|String| |
+|standardPackageType|Standard name used to designate the package type|String| |
+
+#### 3.9	Passive Components Specifications
+The passive components digital datasheet specifications are included in GitHub: https://github.com/edatasheets/edatasheets.github.io
+
+#### 3.9.1	Resistor Specifications
+The table below give a description of the properties used to specify a resistor in a digital datasheet.
+|Property|Description|Data Type|Required?|
+|:----|:----|:----|:----|
+|Value|Resistance value|Number|Yes|
+|Tolerance|Nominal tolerance of a resistor|Number| |
+|powerRating|Maximum power a resistor can dissipate without degrading performance|Number| |
+|temperatureCoefficient|Change in resistance when the temperature is changed|Number| |
+|maxOverloadVoltage|Maximum voltage that can be applied to the resistor for a short period of time|Number| |
+|maxLimitingElementVoltage|Maximum voltage value that can be applied continuously to the resistor.|Number| |

```

Comment:
I'd remove the . here (applied continuously to the resistor) to be consistent with other lines in the table

Reply:
Done